networkx/algorithms/centrality/domirank.py

```diff
@@ -0,0 +1,305 @@
+"""DomiRank Centrality: By Marcus Engsig (@mengsig)"""
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+__all__ = ["domirank"]
+
+
+@not_implemented_for("multigraph")
+@nx._dispatchable(edge_attrs="weight")
+def domirank(
+    G, analytical=False, sigma=0.95, dt=0.1, epsilon=1e-5, max_iter=1000, check_step=10
+):
+    r"""Compute the DomiRank centrality for the graph G.
+
+    DomiRank centrality computes the centrality for a node by adding
+    1 minus the centrality of its neighborhood. This essentially finds the
+    dominance of a node in its neighborhood, where the parameter $\sigma$ determines
```

Comment:
I might be overthinking this, but I don't know if we should perhaps clarify that `$\sigma$` refers to `sigma`.